setup.py

```diff
@@ -24,10 +24,12 @@
 except:
     requirements.append("hashlib")
 
+import bagit
```

Comment:
I believe the PEP-8 convention is to have all the "plain" imports at the top followed by the "from X import Y" declarations